portfolio/src/main/webapp/comments.js

```diff
@@ -58,6 +63,20 @@ class Comments {
     commentElement.innerText = text;
     return commentElement;
   }
+
+  /**
+   * Deletes all comments in DataStore by doing a POST to /delete-comments-data.
+   * Then, the comments are reloaded to confirm the database has been cleared.
+   */
+  deleteComments() {
+    const request = new Request('/delete-comments-data', {method:'POST'})
+    fetch(request)
+    .catch(() => {
```

Comment:
Try to keep the ordering of .then and .catch consistent in your code. If you do then before catch, do that everywhere.